### DISCUSSION TITLE
"entity_category" in Home Assistant

### DISCUSSION THREAD

**Discussion by phil11c** [emsesp/EMS-ESP32#1492] [General]
Since my Update from 3.5x to 3.6.4 all entities in HA are not shown on my autogenerated "Overview" Dashboard.

First thought i did sth. wrong - but reason seems to be, that all "new" entities from 3.6.4 are
```"entity_category": "config"``` or ```"entity_category": "diagnostic"``` and those are [excluded](https://developers.home-assistant.io/blog/2021/10/26/config-entity/#entity-categories)

Why are no entities uncategorized ```entity_category": null``` which leads to autoclassification as "Controls", "Sensors" and so on?
Other devices use "diagnostic" for sensors regarding the device itself (i.E. internal temperature), not what it is made/used for. Same for "config". And btw: those new *shower*-entities are the only ones without an entity_category....

Is there a special reason for it and is there a way for me to change this?
Thanks in advance

---

BBQKees S32 with 3.6.4
HA Core 2023.12.1
HA Supervisor 2023.11.6
HA Operating System 11.2
HA Frontend 20231208.2
Mosquitto broker 6.4.0

**Comment by proddy** [comment]
See https://github.com/emsesp/EMS-ESP32/discussions/1459

this was a long debate. I've changed it 3 times now. Try out the latest dev

**Reply by phil11c** [reply]
thanx a lot
